libs/portal-integration-angular/testing/data-table.harness.ts

```diff
@@ -30,4 +30,19 @@ export class DataTableHarness extends ContentContainerComponentHarness {
       return checkBoxHarnesses
     }
   }
+
+  async getActionColumnHeader(position: 'left' | 'right') {
+      return await this.locatorForOptional(`[name="action-column-header-${position}"]`)()
+  }
+
+  async getActionColumn(position: 'left' | 'right') {
+    return await this.locatorForOptional(`[name="action-column-${position}"]`)()
+  }
+
+  async columnIsFrozen(column: TestElement | null) {
```

Comment:
Is frozen the same as sticky? If yes please make it consistent

Reply:
I renamed everything from sticky to frozen because this terminology is also used in PrimeNG.